pkg/cluster/internal/providers/podman/util.go

```diff
@@ -115,3 +115,19 @@ func deleteVolumes(names []string) error {
 	cmd := exec.Command("podman", args...)
 	return cmd.Run()
 }
+
+// mountDevMapper checks if the podman storage driver is Btrfs or ZFS
+func mountDevMapper() bool {
+	storage := ""
+	cmd := exec.Command("podman", "info", "-f",
+		`{{ index .Store.GraphStatus "Backing Filesystem"}}`)
```

Comment:
we need to get the docker driver checking the backing filesystem too (different PR)